cybersyn/locale/ru/base.cfg

```diff
@@ -112,7 +112,7 @@ is-stack-description=Порог в ячейках
 is-stack-tooltip=При включении параметра все пороговые значения запросов для этой станции интерпретируются как количество полных ячеек, а не общее количество предметов. Не влияет на интерпретацию порогового значения для жидкостей.
 enable-inactive-description=Условие бездействия
 enable-inactive-tooltip=Если параметр включен, поезд на станции снабжения будет выжидать период бездействия, даже если его запрос был выполнен. Это часто бывает полезно для предотвращения застревания предметов в манипуляторах.
-use-same-depot-description=Прикреплять поезд к депо
+use-same-depot-description=Прикреплять поезд к конкретной станции депо
```

Comment:
Isn't this a little long for a UI label? The tooltip explains what the option does.